mandatory/2-function-creation.js

```diff
@@ -25,7 +33,15 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
```

Comment:
Good use of let. There's one var hidden though.

Could you reformat the code inside the for loop to make it more readable?